docs/content/functioneel/zaaktypen_api.md

```diff
@@ -0,0 +1,182 @@
+|----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Aanleiding     | Een consumer wil een lijst van alle beschikbare CATALOGUSsen of een specifieke catalogus opvragen.         |
+| *Toevoegen*    |      |
+| - voorwaarde   | nvt.         |
+| - gevolg       | nvt.             |
+| *Wijzigen*     |      |
+| - voorwaarde   | nvt.         |
+| - gevolg       | nvt.             |
+| *Verwijderen*  |      |
+| - voorwaarde   | nvt.         |
+| - gevolg       | nvt.             |
+| *Raadplegen*   |      |
+| - voorwaarde   | nvt.         |
+| - gevolg       | Een lijst met beschikbare catalogussen wordt teruggegeven aan de consumer bij het opvragen van de lijst.<br>Een specificieke catalogus wordt teruggegeven aan de consumer. |
+| Gegevens       | objecttype CATALOGUS inclusief een lijst met relaties naar daarin gedefinieerde ZAAKTYPEn.<br>Zie de volgende tabel voor de gegevens van deze resource                                 |
+| Specificaties  | https://ref.tst.vng.cloud/ztc/api/v1/schema/#tag/catalogus                                                                                                   |
+| Samenhang      | nvt. |
+| Bijzonderheden | nvt. |
+        
+
+
+### Resource: zaaktype
+
+| **Aspect**     | **Beschrijving**                                                                                                                                                                 |
+|----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Aanleiding     | Een consumer wil (een lijst van beschikbare) zaaktype informatie opvragen uit de catalogus.            |
+| *Toevoegen*    |      |
+| - voorwaarde   | nvt.         |
+| - gevolg       | nvt.             |
+| *Wijzigen*     |      |
+| - voorwaarde   | nvt.         |
+| - gevolg       | nvt.             |
+| *Verwijderen*  |      |
+| - voorwaarde   | nvt.         |
+| - gevolg       | nvt.             |
+| *Raadplegen*   |      |
+| - voorwaarde   | Om zaaktype informatie op te kunnen vragen moet de consumer de UUID van de catalogus waarin het zaaktype is opgenomen weten. Deze UUID kan opgevraagd worden met de catalogus bevragingen services.       |
+| - gevolg       | Een lijst met in de catalogus opgenomen zaaktypen wordt teruggegeven aan de consumer bij het opvragen van de lijst.<br>Een specificiek zaaktype wordt teruggegeven aan de consumer.         |
+| Gegevens       | ZAAKTYPE inclusief STATUSTYPEn, EIGENSCHAPpen en ROLTYPEn<br>Zie de volgende tabel voor de gegevens van deze resource                                 |
+| Specificaties  | https://ref.tst.vng.cloud/ztc/api/v1/schema/#tag/zaaktype                                                                                |
```

Comment:
deze URL werkt niet

Reply:
Dat was mij ook al opgevallen, hoe krijgen we dit wel werken? In Redoc kan ik veel dingen niet die ik met swaggerhub wel kan. Dat kan mijn gebrek aan kennis zijn of mist er iets aan de OAS?

Reply:
Heeft niets met de OAS te maken (Redoc of Swaggerhub zijn maar grafische layouts om dezelfde OAS). Wel met de prioriteiten van Redoc features, waar dit punt laag op de agenda staat. 

Er vragen meer mensen om, zie: https://github.com/Rebilly/ReDoc/issues/134

Reply:
Ok, voor nu is het wel belangrijk dat we dit oplossen/omzeilen. VNG heeft een Swaggerhub omgeving beschikbaar, kunnen we daar dingen wel die niet met Redoc kunnen? Dan stel ik voor de best geschikte tooling te gebruiken en ons niet te laten beperken omdat een leverancier iets niet ingebouwd heeft :-)

Reply:
Maak er een User Story voor aan :)

> Dan stel ik voor de best geschikte tooling te gebruiken en ons niet te laten beperken omdat een leverancier iets niet ingebouwd heeft :-)

Uiteraard willen we de best geschikte tool. Maar voor wie? Het is een kwestie van rol/perspectief.

Voor mij voldoet bijvoorbeeld: https://ref.tst.vng.cloud/ztc/api/v1/schema/#operation/zaaktype_read

> VNG heeft een Swaggerhub omgeving beschikbaar, kunnen we daar dingen wel die niet met Redoc kunnen

Ik merk enige drang om naar een betaalde omgeving te switchen. Dat is niet nodig. SwaggerUI is gewoon een beschikbaar schilletje om een OAS. We kunnen het zo aanzetten dat beide beschikbaar zijn.

Dus, why not both?

Reply:
> Voor mij voldoet bijvoorbeeld: https://ref.tst.vng.cloud/ztc/api/v1/schema/#operation/zaaktype_read

Voor mij ook. Dit is 'm, in dit geval.

Reply:
> 
> > Dan stel ik voor de best geschikte tooling te gebruiken en ons niet te laten beperken omdat een leverancier iets niet ingebouwd heeft :-)
> 
> Uiteraard willen we de best geschikte tool. Maar voor wie? Het is een kwestie van rol/perspectief.

Dat is wat mij betreft de tool die het werk van ieder het beste en op de eenvoudigste manier ondersteunt.


> 
> Voor mij voldoet bijvoorbeeld: https://ref.tst.vng.cloud/ztc/api/v1/schema/#operation/zaaktype_read

Ziet er goed uit, is er ergens een beschrijving hoe ik deze url's op kan vragen? Ik denk nu wel te snappen hoe ik ze zelf kan afleiden maar als er ergens een slimme truc is om het gegarandeerd goed te doen leer ik die graag.

> 
> > VNG heeft een Swaggerhub omgeving beschikbaar, kunnen we daar dingen wel die niet met Redoc kunnen
> 
> Ik merk enige drang om naar een betaalde omgeving te switchen. Dat is niet nodig. SwaggerUI is gewoon een beschikbaar schilletje om een OAS. We kunnen het zo aanzetten dat beide beschikbaar zijn.

Het is geen drang, meer gebrek aan kennis en de beschikking hebben over een tool die dat gebrek lijkt te kunnen compenseren.

> 
> Dus, why not both?

Vind ik prima, mag van mij ook gerust alleen redoc zijn alleen wil dan graag alles kunnen wat ik met swaggerhub kan en weten hoe dat moet (vooral dat laatste vermoed ik).

Reply:
> 
> 
> > Voor mij voldoet bijvoorbeeld: https://ref.tst.vng.cloud/ztc/api/v1/schema/#operation/zaaktype_read
> 
> Voor mij ook. Dit is 'm, in dit geval.

Eigenlijk is dit hem niet: de Resource Zaaktype lijkt nu beschreven in het kader van de lees-operatie. Terwijl het schema van Zaaktype uiteindelijk ook voor (de nog toe te voegen) muteer-operaties zal gaan gelden (verwacht ik). Ik zou hier dus een link naar het schema van het objecttype verwachten en niet naar de operatie zaaktype_read.

Reply:
Het zou m.i. een link moeten zijn naar de resource (zoiets als https://ref.tst.vng.cloud/ztc/api/v1/schema/#tag/zaaktypen) maar die resource lijkt niet onderscheiden te worden.

Reply:
precies wat ik bedoel!